build-configs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "build-configs",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Do we really need to push the version here always? Is this because it is not symlinked?

Reply:
Yes, probably because of missing symlinks.